src/fswAlgorithms/imageProcessing/centerOfBrightness/centerOfBrightness.cpp

```diff
@@ -129,3 +129,43 @@ void CenterOfBrightness::UpdateState(uint64_t CurrentSimNanos)
     return;
 }
```

Comment:
:small_red_triangle_down: I'd squash this commit into the previous one. They go together.

Reply:
Done